language-plutus-core/src/Language/PlutusCore/Lexer/Type.hs

```diff
@@ -49,7 +50,7 @@ data BuiltinName = AddInteger
                  | TxHash
                  | BlockNum
                  | BlockTime
-                 deriving (Show, Eq, Ord, Generic, NFData, Lift)
```

Comment:
Do we plan to remove the last two built-ins?

Reply:
BlockTime is no longer in the spec so you can safely remove it.

Reply:
What about `BlockNum`?

Reply:
I think, we decided to keep `blocknum` as a proxy for blockchain information provided to validator code. (In concrete uses of PLC on the blockchain, there will be further builtins, but they depend on the particular chain.)